src/docs/quicktips/internal-js.md

```diff
@@ -64,5 +64,5 @@ Capture the JavaScript into a variable and run it through the filter (this sampl
 ### Warning about Content Security Policy
 
 {% callout "warn" %}
-If you are using a Content Security Policy on your website, make sure the <code>script-src</code> directive allows <code>'unsafe-inline'</code>. Otherwise, your inline Javascript will not load.
+If you are using a Content Security Policy on your website, make sure the <code>script-src</code> directive allows <code>'unsafe-inline'</code>. Otherwise, your internal Javascript will not load.
```

Comment:
```suggestion
If you are using a Content Security Policy on your website, make sure the <code>script-src</code> directive allows <code>'unsafe-inline'</code>. Otherwise, your internal JavaScript will not load.
```